### PR TITLE
More args

### LINKED ISSUE #33: Allow client to specify command line args
Update client to use flags to clean up argument parsing

### PR DESCRIPTION
**Pull request by tuckerzp**
Change how command line works:
**./client {optional flags} host file** --- do ./client -h for help with optional flags

One of the optional flags is -args:
examples:
1. ls "-a -t -r"
2. hostname "-I"

### CODE REVIEW

File: cmd/client/main.go [hunk 1]
```diff
@@ -91,6 +61,50 @@ func main() {
 
 /* ----- Helper functions ----- */
 
+func parseCommandLine() (string, string, int64, int64, []string, int64) {
+	// Optional flags
+	argsPtr := flag.String("args", "NONE", "Command line args for file")
+	rangePtr := flag.String("range", "NONE", "Range for job")
```
Comment (showalter) [inline cmd/client/main.go]: Provide an example for this one especially

**Review by showalter — CHANGES_REQUESTED** [review]
Looks good! Just look at this one thing

File: cmd/client/main.go [hunk 2]
```diff
@@ -19,40 +20,9 @@ import (
...
-		}
-
-		start, err = strconv.ParseInt(split[0], 10, 64)
-		check(err)
-
-		end, err = strconv.ParseInt(split[1], 10, 64)
-		check(err)
-	}
-
+	hostName, fullFileName, start, end, args, runs := parseCommandLine()
```
Comment (showalter) [inline cmd/client/main.go]: One more thing, could we consider passing these things in by pointer to parseCommandLine? This many returns is kind of gross.

File: cmd/client/main.go (hunk 1 above)
Comment (tuckerzp) [reply cmd/client/main.go]: for the range or the args?

File: cmd/client/main.go (hunk 2 above)
Comment (tuckerzp) [reply cmd/client/main.go]: Yeah I was thinking the same thing

File: cmd/client/main.go (hunk 1 above)
Comment (showalter) [reply cmd/client/main.go]: I think having an example for both would be ideal

**Review by showalter — APPROVED** [review]

**Pull request merged**